branchtesting/src/main/java/branchexample/App.java

```diff
@@ -4,12 +4,12 @@
 {
     public void aMethod( String[] args )
     {
-    	// TODO fix this
     	int i = 0;
-    	if (i == 0) {
+    	if (i != 0) {
     		// TODO fix this too..
     		int j = 5 / i;
     	}
+    	//TODO make this a logger
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://sonarcloud.io/organizations/brian-cipollone-sonarsource-gith/rules#rule_key=squid%3AS1135))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.example.bcipollone%3Abranchtesting&pullRequest=22&issues=AWaH3AUulB6-zLfzr5-D&open=AWaH3AUulB6-zLfzr5-D)

[ISSUE_KEY]: AWaH3AUulB6-zLfzr5-D